src/sortStudents.ts

```diff
@@ -1,16 +1,94 @@
+/* eslint-disable no-debugger */
+/* eslint-disable no-console */
+/* eslint-disable max-len */
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(students: Student[], sortBy: SortType, order: SortOrder): Student[] {
```

Comment:
It's hard to read it now 
```suggestion
export function sortStudents(
  students: Student[], 
  sortBy: SortType, 
  order: SortOrder
): Student[] {
```